features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -149,6 +151,16 @@ data class SessionReplayConfiguration internal constructor(
             return this
         }
 
+        /**
+         * This option controls whether optimization is enabled or disabled for recording Session Replay data.
+         * When optimization is enabled, a time quota is set per second, which limits the amount of data that can
```

Comment:
Here we're referring to a specific optimization. In the future we'll disable devices that have low specs and potentially other improvements, so will we want to list them all here, or to be more ambiguous about what optimizations we are performing?

Reply:
I think we can be more ambiguous instead of listing them all, because currently we haven't implemented all the improvements yet.

Reply:
updated